pkg/performanceprofile/cmd/render/render.go

```diff
@@ -204,6 +207,8 @@ func render(inputDir, outputDir string) error {
 
 		for _, componentObj := range components.ToObjects() {
 			componentObj.SetOwnerReferences(or)
+			anno := util.AddBootstrapRenderedByAnnotation(componentObj.GetAnnotations(), pp.Name, pp.Namespace)
```

Comment:
I think I would remove this too to make this PR solely about the MCPs.

Reply:
Sure thing, removed